src/main/java/org/prgrms/prgrmsspring/domain/VoucherType.java

```diff
@@ -0,0 +1,58 @@
+package org.prgrms.prgrmsspring.domain;
+
+import org.prgrms.prgrmsspring.entity.voucher.FixedAmountVoucher;
+import org.prgrms.prgrmsspring.entity.voucher.PercentDiscountVoucher;
+import org.prgrms.prgrmsspring.entity.voucher.Voucher;
+import org.prgrms.prgrmsspring.exception.ExceptionMessage;
+import org.prgrms.prgrmsspring.exception.NotFoundException;
+
+import java.util.Arrays;
+import java.util.UUID;
+import java.util.function.BiFunction;
+import java.util.function.Predicate;
+
+public enum VoucherType {
+    FIXEDAMOUNT("FixedAmount", FixedAmountVoucher::new, FixedAmountVoucher.class::isInstance),
+    PERCENTDISCOUNT("PercentDiscount", PercentDiscountVoucher::new, PercentDiscountVoucher.class::isInstance);
```

Comment:
```suggestion
    FIXED_AMOUNT("FixedAmount", FixedAmountVoucher::new, FixedAmountVoucher.class::isInstance),
    PERCENT_DISCOUNT("PercentDiscount", PercentDiscountVoucher::new, PercentDiscountVoucher.class::isInstance);
```

언더스코어를 포함해주시는게 상수를 읽을 때 가독성이 좋을 것 같습니다.

Reply:
넵 반영하겠습니다.